### ISSUE TITLE
IphoneX FastImage performance

### ISSUE THREAD

**Issue by mmamoyco** [DylanVann/react-native-fast-image#605]
**Describe the bug**
FastImage component causes little performance drops (transitions is not smooth enough).
I have 2 screens with one image on each. When i perform transition i see some transition freezes (not critical). But when i remove FastImage from scene - transitions become butter smooth.

what is really interesting - this drop occurs only on > IphoneX devices. Maybe it's some fs interactions issue, idk.

My images is optimized and minified.

**To Reproduce**
Steps to reproduce the behavior if possible, or a link to a reproduction repo:
1. Create two scenes (i use react-navigation)
2. Add Fastimage on each scene
3. Perform transition
4. See micro transition freezes

**Expected behavior**
Clear and butter smooth transition

**Dependency versions**

- React Native version: ^0.59.10
- React version: ^16.8.6
- React Native Fast Image version:^6.0.2

**Comment by mrousavy** [comment]
experiencing the same right here on an iPhone 11 Pro. Default `<Image>` component from `react-native` performs way better. 

The `<FastImage>` component looks a little pixelated, and re-renders these pixels on transitions instead of just smoothing the image like the `<Image>` component. In other words; The `<Image>` component looks slightly blurred instead of high-detail-pixelated, and I think that's what's causing the performance drops. Hopefully that explanation was understandable..

Anyone knows why this is only happening on iPhone X and above?

**Comment by mrousavy** [comment]
Since this issue is really old, I'd like to bump this.

Noticeable performance lags still occur on 
```
"react-native-fast-image": "^8.1.5",
"react-native": "0.62.2",
"@react-navigation/native": "^5.4.0",
"@react-navigation/stack": "^5.3.7",
```

**Comment by manuhook** [comment]
Hi, @mrousavy since there is no answer on this issue, did you find a good alternative to cache images on React Native ?
On iOS with a large flatlist and the original <Image> component I don't have good results with cache:"force-cache" :-/

**Comment by mrousavy** [comment]
@manuhook did you actually compare the native <Image> component to the <FastImage> component? Was FastImage noticeably slower for you?

**Comment by manuhook** [comment]
@mrousavy yes same transition issue as described here, however it's seems to perform better when using react-native-navigation

**Comment by mrousavy** [comment]
@manuhook I'd love to use react-native-navigation as well, awesome work from the guys at wix. Unfortunately I've got the Apple's app of the day shared element transition in my app, which I can't replicate with the current capabilities of react-native-navigation yet :/

Also to answer your question; No, I didn't find a better solution to this. You can either use `<FastImage>` or `<Image>` with force-cache.

**Comment by ghasemikasra39** [comment]
The performance on ios is terrible, only helps on Android. On ios you may want to use the built-in `Image` component from React Native.
